src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -78,10 +79,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-        sb.append("Gui Settings : " + guiSettings);
-        sb.append("\nLocal data file location : " + recipeBookFilePath);
-        return sb.toString();
+        return String.format("GUI Settings : %s \nLocal data file location : %s", guiSettings, recipeBookFilePath);
```

Comment:
just a minor thing but I think using the StringBuilder way makes it easier to read.

Reply:
OKay 😋 , i'll change it to stringbuilder69 👍 🔥